### PR TITLE
Responsive: Typography & Details

### PR DESCRIPTION
**Pull request by varffs**
doesnt adequately solve header issues that is for another branch

### CODE REVIEW

**Review by mikeyrayvon — COMMENTED** [review]
the mobile layout...confuses me..

**Comment by mikeyrayvon** [comment]
screen shot 2017-05-18 at 12 46 57 pm
that linebreak is wrong no?

**Comment by mikeyrayvon** [comment]
screen shot 2017-05-18 at 12 47 56 pm
😐

File: css/responsive/desktop.styl [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+// TYPOGRAPHY
+
+body
+  font-size: $font-size-basic
+  line-height: $font-leading-basic
+  letter-spacing: $font-tracking-basic
+
+.font-leading-wide
+  line-height: $font-leading-wide
```
Comment (mikeyrayvon) [inline css/responsive/desktop.styl:14]: wheres this var come from

Comment (varffs) [reply css/responsive/desktop.styl:14]: site.styl

**Comment by varffs** [comment]
yh

>doesnt adequately solve header issues that is for another branch

But the linebreak on the colon is wrong and now fixed

**Pull request merged**